test/reporters/xunit.spec.js

```diff
@@ -194,15 +194,13 @@ describe('XUnit reporter', function() {
       it('should call write with line', function() {
         stdoutWrite = process;
         process = false; // eslint-disable-line no-native-reassign, no-global-assign
-        var cachedConsoleLog = console.log;
-        console.log = function(string) {
-          stdout.push(string);
-        };
 
+        var cachedConsoleLog = reporters.Base.consoleLog;
```

Comment:
nit: this would seem better named as `savedConsoleLog` -- not what I think about when I see the word "cached"...